src/sql/workbench/browser/modal/modal.ts

```diff
@@ -391,14 +391,15 @@ export abstract class Modal extends Disposable implements IThemable {
 	/**
 	 * Hides the modal and removes key listeners
 	 */
-	protected hide(reason?: string) {
+	protected hide(reason?: string, currentPageName?: string) {
```

Comment:
While you're here if you could add the return type (void) to this function that'd be great